modules/eks-managed-node-group/main.tf

```diff
@@ -238,6 +238,7 @@ resource "aws_launch_template" "this" {
 
   lifecycle {
     create_before_destroy = true
+    ignore_changes = var.launch_template_ignore_changes
```

Comment:
lifecycle blocks do not accept variables so this won't work

Reply:
oh my god 😢